src/modules/app/AppComponent.js

```diff
@@ -78,7 +78,7 @@ export class AppComponent extends Component {
     if (nodeId) {
       hashHistory.push('/magasin/' + nodeId);
     } else {
-      this.props.clear();
+      this.props.clearPicklist();
```

Comment:
this is malplaced. I really think you should call clearPicklist after call to setCollectionId, and not within the else. What if nodeId is set? Then picklist will not be cleared. Which is  wrong. If collection is changed, and the spec is to clear all nodes and all objects on collection change, it is correct to clear. But, I think its not relevant to clear nodes in picklist on collection change. Its only important to clear objects on collection change